source/tracking/tracking_main.py

```diff
@@ -36,40 +34,14 @@ def init(image, bboxes, video = []):
     # Attempts to start the tracker
     ok = tracker.init(image, bbox)
     print(ok)
-    # Checks if the initialization was successful
-    if ok:
-        # Goes through each frame that occurred since the first image was found
-        for frame in video:
-            ok = draw(frame)
-
-    # Returns the tracker's status
-    return ok
-
+    
+    # returns the tracked bounding box if tracker was successful, otherwise None
+    return bbox if ok else None
 
 # Updates the location of the object
 def update(image):
     # Attempts to update the object's location
     ok, location = tracker.update(image)
 
-    # Returns the location if the location was updated
-    if ok:
-        # Return centerpoint location (x, y)
```

Comment:
Dang did we really not need any of this?

Reply:
I changed the return and input parameters so that's why I'm returning the location with the width and height and from what I saw we never used draw anywhere and it seemed unnecessary so I thought there was no point in even keeping it.

Reply:
Great thanks for cleaning it out 👍